### PR TITLE
Use box center as reference point everywhere

### PR DESCRIPTION
**Pull request by akretz**
this fixes opendatacam/opendatacam#410

### CODE REVIEW

File: main.js
Comment (vsaw) [inline main.js:143]: Shouldn't we subtract the width and height instead of adding it? Or am I misunderstanding the coordinate system?

**Review by vsaw — CHANGES_REQUESTED** [review]

File: main.js
Comment (akretz) [reply main.js:143]: It should be correct that way. The format of the MOT17 input file is
`<frame>, <id>, <bb_left>, <bb_top>, <bb_width>, <bb_height>, <conf>, <x>, <y>, <z>`
so we're going from top left corner to center point, which means adding half of width and height.

Comment (vsaw) [reply main.js:143]: Sorry, you're right! 👍

**Review by vsaw — APPROVED** [review]

**Comment by vsaw** [comment]
This looks good to me but we will have to wait for @tdurand as I can't merge code in this repo :-)

**Comment by tdurand** [comment]
now you do @vsaw , feel free to merge, thanks for the PR.. 

sorry I don't have the bandwith to review / process this week

**Comment by tdurand** [comment]
+ vsaw send me your npmjs.com account by email so you can release a new version on the package once merged..

**Comment by vsaw** [comment]
I tested it locally. Looks good (see attached video)

https://user-images.githubusercontent.com/829308/123399322-c4eab100-d5a4-11eb-800a-1ae32b21aa45.mp4

Will merge and release a new version of the tracker thank you @akretz for finding and fixing this bug!

**Pull request merged**
